### PR TITLE
Use translated welcome emails

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Uses the translated welcome emails! 🥳 

Screenshot 2024-07-16 at 12 02 49 PM

I've created [a task](https://codedotorg.atlassian.net/browse/ACQ-2071) to clean up more of the code in a follow-up.

### CODE REVIEW

File: lib/test/cdo/test_mailjet.rb [hunk 1]
```diff
@@ -89,7 +91,46 @@ def test_send_template_email
         messages[0][:TemplateID] == template_id
     end
 
-    MailJet.send_template_email(mock_contact, email_config)
+    MailJet.send_template_email(mock_contact, email_config, 'en-US')
+  end
+
+  def test_send_template_email_locoalized
```
Comment (TurnerRiley) [inline lib/test/cdo/test_mailjet.rb]: Nit: spelling of localized

**Review by TurnerRiley — APPROVED** [review]
LGTM! Out of curiosity, will Mailjet handle any translations? And if so, will we then have to retroactively add in the template ids down below for different locales?

**Comment by bethanyaconnor** [comment]
> Out of curiosity, will Mailjet handle any translations? And if so, will we then have to retroactively add in the template ids down below for different locales?

There's a MailJet <-> Crowdin integration that we're utilizing. We would need to add template IDs for different locales if we decided to use the translations for those locales.

File: lib/test/cdo/test_mailjet.rb (hunk 1 above)
Comment (bethanyaconnor) [reply lib/test/cdo/test_mailjet.rb]: woops thank you!

**Pull request merged**